pkg/occlient/occlient_test.go

```diff
@@ -1230,7 +1230,7 @@ func TestCleanupAfterSupervisor(t *testing.T) {
 					}
 				}
 			} else if err == nil && tt.wantErr {
-				t.Errorf("test failed, expected: %s, got %s", "error", "no error")
+				t.Error("test failed, expected: error, got no error")
```

Comment:
When you see just this error it is not clear what happened.
"error was expected, but no error returned" or something like this